crates/turborepo-lockfiles/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(trait_upcasting)]
```

Comment:
Required in order to cast `&dyn Lockfile` to `&dyn Any` even through `Lockfile: Any` (See [supertraits](https://doc.rust-lang.org/rust-by-example/trait/supertraits.html))

(This is only a gated feature at the moment because it does have effect the vtable memory size, but it has been deemed minor enough that it will be [opt-out in the future](https://github.com/rust-lang/rust/issues/65991))